lib/ClimaCoreTempestRemap/src/wrappers.jl

```diff
@@ -0,0 +1,30 @@
+function rll_mesh(filename::String; nlat = 90, nlon = round(Int, nlat * 1.6))
+    run(
```

Comment:
I think we should use `pipeline` here so we can redirect `stdout`, `stderr` to `devnull` if `verbose=false` keyword is set (default).